modules/osd-monitoring-specifying-how-a-service-is-monitored.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-managing-metrics.adoc
+
+[id="specifying-how-a-service-is-monitored_{context}"]
+= Specifying how a service is monitored
+
+To use the metrics exposed by your service, you must configure {product-title} monitoring to scrape metrics from the `/metrics` endpoint. You can do this using a `ServiceMonitor` custom resource definition (CRD) that specifies how a service should be monitored, or a `PodMonitor` CRD that specifies how a pod should be monitored. The former requires a `Service` object, while the latter does not, allowing Prometheus to directly scrape metrics from the metrics endpoint exposed by a pod.
+
+[NOTE]
+====
+In {product-title}, you can use the `tlsConfig` property for a `ServiceMonitor` resource to specify the TLS configuration to use when scraping metrics from an endpoint. The `tlsConfig` property is not yet available for `PodMonitor` resources. If you need to use a TLS configuration when scraping metrics, you must use `ServiceMonitor` resource.
+====
+
+This procedure shows you how to create a `ServiceMonitor` resource for a service in a user-defined project.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `dedicated-admin` role or the `monitoring-edit` role.
+* For this example, you have deployed the `prometheus-example-app` sample service in the `ns1` project.
+
+.Procedure
+
+. Create a YAML file for the `ServiceMonitor` resource configuration. In this example, the file is called `example-app-service-monitor.yaml`.
+
+. Add the following `ServiceMonitor` resource configuration details:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: prometheus-example-monitor
+  name: prometheus-example-monitor
+  namespace: ns1
+spec:
+  endpoints:
+  - interval: 30s
+    port: web
+    scheme: http
+  selector:
+    matchLabels:
+      app: prometheus-example-app
+----
++
+This defines a `ServiceMonitor` resource that scrapes the metrics exposed by the `prometheus-example-app` sample service, which includes the `version` metric.
+
+. Apply the configuration to the cluster:
++
+[source,terminal]
+----
+$ oc apply -f example-app-service-monitor.yaml
+----
++
+It takes some time to deploy the `ServiceMonitor` resource.
+
+. You can check that the `ServiceMonitor` resource is running:
++
+[source,terminal]
+----
+$ oc -n ns1 get servicemonitor
+----
++
+.Example output
+[source,terminal]
+----
+NAME                         AGE
+prometheus-example-monitor   81m
+----
+
+.Additional resources
+
+* See the link:https://github.com/openshift/prometheus-operator/blob/release-4.5/Documentation/api.md[Prometheus Operator API documentation] for more information on `ServiceMonitor` and `PodMonitor` resources.
```

Comment:
link is wrong, if OSD release version is 4.6, should link to https://github.com/openshift/prometheus-operator/blob/release-4.6/Documentation/api.md

Reply:
I changed it to 4.7, since OSD now supports 4.7. Is this correct now?